pages/common/pgrep.md

```diff
@@ -1,14 +1,15 @@
 # pgrep
 
 > Find or signal process by name.
+> More information: <https://www.man7.org/linux/man-pages/man1/pkill.1.html>.
 
 - Return PIDs of any running processes with a matching command string:
 
 `pgrep {{process_name}}`
 
 - Search full command line with parameters instead of just the process name:
```

Comment:
```suggestion
- Search for processes, specifying its command line options:
```
Maybe this will read a little better?

Reply:
I prefer something like: `Search for a process name including the command line options`, or similar.

Reply:
Sure, but I think we should still use the plural, since we search for all matching processes.

Reply:
```suggestion
- Search for processes including their command line options:
```